.NET/Microsoft.Recognizers.Definitions.Common/English/TimeZoneDefinitions.cs

```diff
@@ -188,6 +189,7 @@ public static class TimeZoneDefinitions
             @"Australian Central Time",
             @"AUS Eastern Standard Time",
             @"Australian Eastern Time",
+            @"Australian Eastern Standard Time",

```

Comment:
Can you also add:
Australian Central Daylight Time or ACDT has a UTC offset of +10:30.
Australian Eastern Daylight Time or AEDT has a UTC offset of + 11:00.
Please make sure "Eastern Daylight Time" and "Central Daylight Time"have offset as -1000 as they are ambiguous between the US and Australia.